src/components/Panel/Panel.js

```diff
@@ -29,6 +46,28 @@ class Panel extends Component {
           />
         )}
 
+        {collapsible && (
+          <button
+            className={cssClasses.collapseButton}
+            aria-expanded={!this.state.collapsed}
+            onClick={event => {
+              event.preventDefault();
+
+              this.setState(previousState => ({
+                controlled: true,
+                collapsed: !previousState.collapsed,
+              }));
+            }}
+          >
+            <Template
+              {...templateProps}
+              templateKey="collapseButton"
```

Comment:
IIRC we went for `xxxText` in the other template no?

Reply:
Indeed, we went for [`showMoreText`](https://github.com/algolia/instantsearch.js/blob/9f2aac2d231456af99d40d8719194beddf8ffa2a/src/components/RefinementList/RefinementList.js#L161) for `refinementList`. The main difference is that here it's a `span` and not a `button`.

It could be a bit misleading to call it `xxxText` because it can be something else than just text (just like in a `button` actually).

We can go for `xxxText` for consistency. I'm not sure it's very appropriate though.